services/webinar/utils.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import fetch, { RequestInit } from 'node-fetch';
+
+export const apiBase = 'https://api.getgo.com';
+
+export function fetchWithBody(
```

Comment:
I'm curious about this having to be a custom function? It seems like a pretty standard thing to want in terms of functionality... or is it the use of Promises that makes this 'special'? 
Disclaimer: I'm somewhat new to the wider JS ecosystem so not sure at all about what to expect from the language :)